### ISSUE TITLE
OpenXRay Linux build - xrScriptEngine error

### ISSUE THREAD

**Issue by McSeem** [OpenXRay/xray-16#375]
Build error with script_engine.cpp file occurs:
```
src/xrScriptEngine/script_engine.cpp: In member function ‘void CScriptEngine::init(CScriptEngine::ExporterFunc, bool)’:
/home/mcseem/Builds/xray-16/src/xrScriptEngine/script_engine.cpp:999:18: error: ‘allow_nil_conversion’ is not a member of ‘luabind’
         luabind::allow_nil_conversion(nilConversion);

```
Kubuntu 18.04.1
CPU: Ryzen 7 2700
make version: 4.1

**Comment by Xottab-DUTY** [comment]
I just fixed it with 94b070d6bae426caf2b7ee14be1a9e439e8647fb. Please, let me know if problem isn't fixed..

**Comment by Xottab-DUTY** [comment]
According to [Travis](https://travis-ci.org/OpenXRay/xray-16/builds/510840651) and [AppVeyor](https://ci.appveyor.com/project/OpenXRay/xray-16/builds/23328411) it's fixed.
